internal/repositories/connection.go

```diff
@@ -0,0 +1,39 @@
+package repositories
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"github.com/integrity-sum/internal/core/models"
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+)
+
+func ConnectionToDB(logger *logrus.Logger) (*sql.DB, error) {
+	connectionDB := models.ConnectionDB{
+		Dbdriver:   os.Getenv("DB_DRIVER"),
+		DbUser:     os.Getenv("DB_USER"),
+		DbPassword: os.Getenv("DB_PASSWORD"),
+		DbPort:     os.Getenv("DB_PORT"),
+		DbHost:     os.Getenv("DB_HOST"),
+		DbName:     os.Getenv("DB_NAME"),
+	}
+	if connectionDB.Dbdriver == "" {
+		logger.Error("DB_DRIVER was not set")
```

Comment:
we can `return errors.New("DB_DRIVER was not set")` here. 
or we can log a little more data and do some actions. like 
`
logger.Errorf("DB_DRIVER was not set. setting by default: %s",  defaultValue)
`

Reply:
or we can validate needed for us env values before creating a connection to DB. On the stage of setting env values to environment

Reply:
https://github.com/ScienceSoft-Inc/integrity-sum/pull/1/commits/ac67a41c7f995d340d28d679575722c33bdf935a#diff-d8b26157f5d098431f11d44d50e4432702510f6044b42dca6390530b90e66377
created new function. It's a normal or not?